### PR TITLE
Web Application Performance, What is Fast?

### PR DESCRIPTION
**Pull request by Tom-Davidson**
Markdown valid as a gist so hoping that it's valid in Jekyll too.

Comments on the actual article contents are welcome too!

### CODE REVIEW

File: internal/_posts/2015-10-29-web-application-performance-what-is-fast.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+image:   http://streamingvectors.com/wp-content/uploads/2014/01/IndiaCarSpeedDialGold32.jpg
+excerpt: "Perceived performance is a key factor in abandonment rate and in the percentage of those abandonments that are permanent. How should we think about performance, measure it and know that we are fast enough?"
+---
+
+# Web Application Performance, What is 'Fast'?
+
+Perceived performance is a key factor in abandonment rate and in the percentage of those abandonments that are permanent. How should we think about performance, measure it and know that we are fast enough?
+
+Performance can be measured as: "The top of the page is visually complete and all elements that are on-screen are fully interact-able." This does not mean that elements further down the page need to be loaded or working in that time nor does it mean purely display elements near the top need to be fully-functional (e.g. carousels) but they do need to be visually complete (e.g. for carousel the first frame must be displayed). Sites perform differently depending on the caching of static assets, as such it’s not as simple as just a single measurement.  I’m considering the following scenarios:
```
Comment (craggy2593) [inline internal/_posts/2015-10-29-web-application-performance-what-is-fast.markdown]: "nor does it mean purely display elements near the top need to be fully-functional"

Slightly confusing in wording to me? 'purely display'?

**Comment by craggy2593** [comment]
:+1:

**Comment by Tom-Davidson** [comment]
@craggy2593 tweaked the wording to make it more clear.

**Comment by sboundy** [comment]
LGTM - nice article.

**Pull request merged**